.eslintrc

```
@@ -114,6 +120,27 @@
         "varsIgnorePattern": "^_",
         "caughtErrorsIgnorePattern": "^_"
       }
+    ],
+    "import/order": [
+      "error",
+      {
+        "groups": ["builtin", "external", "internal"],
+        "pathGroups": [
+          { "pattern": "react", "group": "builtin", "position": "after" }
+        ],
+        "pathGroupsExcludedImportTypes": ["react"],
```

```suggestion
        "pathGroupsExcludedImportTypes": ["react"],
        "newlines-between" : "always",
```

💊이것도 개인취향의 영역인가 싶긴한데 그룹별로 개행 하나씩 넣어두면 보기 편했던거 같아요

---

해보니까 나쁘지 않은 것 같군